client/state/post-types/schema.js

```diff
@@ -0,0 +1,23 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			additionalProperties: {
+				type: 'object',
+				required: [ 'name' ],
+				properties: {
+					name: { type: 'string' },
```

Comment:
This is optional, but if you have a known list of post types, you can be stricter here and add a pattern property.
```javascript
name: { 
     type: 'string',
     pattern: '^(post|page|something_else)$'
}
```